components/alibi-explain-server/tests/make_test_models.py

```diff
@@ -27,7 +27,8 @@ def make_anchor_image(dirname: Optional[Path] = None) -> AnchorImage:
 
     # we drop the first batch dimension because AnchorImage expects a single image
     image_shape = model.get_layer(index=0).input_shape[0][1:]
-    alibi_model = AnchorImage(predictor=model, image_shape=image_shape)
+
+    alibi_model = AnchorImage(predictor=model.predict, image_shape=image_shape)
```

Comment:
Why this change?

Reply:
this is because [call](https://www.tensorflow.org/api_docs/python/tf/keras/Model#call) returns a tensor and alibi `0.6.2` will complain that it doesnt return numpy. [predict](https://www.tensorflow.org/api_docs/python/tf/keras/Model#predict) does return numpy.